### PR TITLE
[LP-1621]Added management command to auto-generate certificates for courses currently running

### PR DESCRIPTION
**Pull request by saad-waqar**
Added management command to auto-generate certificates for courses currently running

### CODE REVIEW

File: common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+"""
+Django management command to auto generate certificates for all users
+enrolled in currently running courses with early_no_info or early_with_info set
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py]: `with early_no_info or early_with_info set` where?

File: common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def handle(self, *args, **options):
+        for course in modulestore().get_courses():
+            if not is_course_valid_for_certificate_auto_generation(course):
+                continue
+
+            for user_course_enrollment in CourseEnrollment.objects.filter(course=course.id).all():
+                user = user_course_enrollment.user
+                cert_data = _get_cert_data(user, course, user_course_enrollment.mode)
+
+                if not cert_data or cert_data.cert_status != CertificateStatuses.requesting:
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py]: I hope this line makes sure that certificate of this user has already not been generated?

File: common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                    continue
+
+                status = generate_user_certificates(user, course.id, course=course)
+                message_f = 'Certificate generation {} for user with username: ' \
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py]: Can we create a constant of this string and apply format method later to insert values

File: common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py (hunk 2 above)
Comment (saad-waqar) [reply common/djangoapps/philu_commands/management/commands/generate_certificates_for_early_no_info.py]: we're checking cert_data.cert_status to be only "requesting"
```
class CertificateStatuses(object):
    """
    Enum for certificate statuses
    """
    deleted = 'deleted'
    deleting = 'deleting'
    downloadable = 'downloadable'
    error = 'error'
    generating = 'generating'
    notpassing = 'notpassing'
    restricted = 'restricted'
    unavailable = 'unavailable'
    auditing = 'auditing'
    audit_passing = 'audit_passing'
    audit_notpassing = 'audit_notpassing'
    honor_passing = 'honor_passing'
    unverified = 'unverified'
    invalidated = 'invalidated'
    requesting = 'requesting'
```

**Pull request merged**